ui/raidboss/data/07-dt/trial/zoraal-ja-ex.txt

```diff
@@ -0,0 +1,122 @@
+# -r 4zpQDykmMGJnX2FW -rf=8
+# Sorry, no `-ii` parameters, I cleaned this up by hand because it was such a mess
+
+hideall "--Reset--"
+hideall "--sync--"
+
+# Forward Half:
+# 937B - Right sword glowing, safe spot is front+right
+# 999B - Left sword glowing, safe spot is front+left
+
+# Forward Half:
+# 937D - Right sword glowing, safe spot is back+left
+# 937E - Left sword glowing, safe spot is back+right
+
+# Arena phase
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+3.2 "--sync--" Ability { id: "93A7", source: "Zoraal Ja" } window 3.2,1
+8.0 "--sync--" StartsUsing { id: "9398", source: "Zoraal Ja" } window 8,8
+12.7 "Actualize" Ability { id: "9398", source: "Zoraal Ja" }
+23.0 "Multidirectional Divide (cast)" Ability { id: "93A2", source: "Zoraal Ja" }
+34.0 "Multidirectional Divide (damage)" Ability { id: "93A3", source: "Zoraal Ja" }
+35.0 "Half Full" #Ability { id: "939E|9380", source: "Zoraal Ja" }
+35.0 "Forward Edge/Backward Edge" #Ability { id: "9972|937F", source: "Zoraal Ja" }
+44.2 "Multidirectional Divide (cast)" Ability { id: "93A2", source: "Zoraal Ja" }
+55.2 "Multidirectional Divide (damage)" Ability { id: "93A3", source: "Zoraal Ja" }
+55.3 "Regicidal Rage" #Ability { id: "993C", source: "Zoraal Ja" }
+
+# Tiles phase
+70.5 "Dawn of an Age" Ability { id: "9397", source: "Zoraal Ja" } window 5,5
+84.5 "Vollok" Ability { id: "9357", source: "Zoraal Ja" }
+94.7 "Sync" Ability { id: "9359", source: "Zoraal Ja" }
+103.6 "Chasm of Vollok" Ability { id: "939A", source: "Zoraal Ja" }
+104.1 "Half Full" Ability { id: "939E|9380", source: "Zoraal Ja" }
+112.9 "Greater Gateway" Ability { id: "9381", source: "Zoraal Ja" }
+121.1 "Blade Warp" Ability { id: "935E", source: "Zoraal Ja" }
+129.3 "Forged Track (cast)" Ability { id: "935F", source: "Zoraal Ja" }
+137.5 "Forged Track (damage)" Ability { id: "939D", source: "Fang" }
+137.6 "Fiery Edge/Stormy Edge" #Ability { id: "938[36]", source: "Fang" }
```

Comment:
Same here, e.g. 'Fiery/Stormy Edge' (and repeated below).  Also per the convention, I think id should be `9383|9366`

Reply:
Yes, as much as possible let's try to expand regexes. It's not impossible to script around it, but the conversions I did alongside quisquous were a lot easier because I had insisted on this for my own work previously.